lightwood/encoders/time_series/helpers/rnn_helpers.py

```diff
@@ -0,0 +1,54 @@
+
+
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
This needs to be:

```
        device_str = "cuda" if CONFIG.USE_CUDA else "cpu"
        if CONFIG.USE_DEVICE is not None:
            device_str = CONFIG.USE_DEVICE
```

This is the default device setting behavior for everything else and it allows the user to specify cpu even if they have a gpu and to specify training on only a specific device (if they have multiple ones).

Reply:
good point